templates/layout.pug

```diff
@@ -5,6 +5,7 @@ html(lang="en")
       include head
       +head()
   body
+    include ../www/svg/coolhole-icons.svg
```

Comment:
question: does this need to be here? if so, should we remove the include from `channel` then?

Reply:
`channel` and `layout` exist on two separate planes, I forget which `.pug` file does this but only one pug extends specifically from `layout`. Adding it to `head` may have caused some issues, but I can't remember exactly at the moment.

Reply:
if you find out, lmk. trying to reduce the amount of changes to common files if possible